src/model.rs

```diff
@@ -24,19 +24,21 @@ impl CxxLogEvent {
         }
     }
 
-    pub fn get_string(&self, key: &str) -> &str {
```

Comment:
we handle invalid utf string by as_str method, which will do replacement of invalid chars(not in-place), so we can not return reference of the str, instead we return the replaced string.